actors/account/src/lib.rs

```diff
@@ -104,28 +91,23 @@ impl Actor {
     }
 
     // Always succeeds, accepting any transfers.
-    pub fn universal_receiver_hook<BS, RT>(
-        rt: &mut RT,
+    pub fn universal_receiver_hook(
+        rt: &mut impl Runtime,
         _params: &RawBytes,
-    ) -> Result<(), ActorError>
-    where
-        BS: Blockstore,
-        RT: Runtime<BS>,
-    {
+    ) -> Result<(), ActorError> {
         rt.validate_immediate_caller_accept_any()?;
         Ok(())
     }
 }
 
 impl ActorCode for Actor {
-    fn invoke_method<BS, RT>(
+    fn invoke_method<RT>(
```

Comment:
Because it shows up in the interface. Basically, if the trait specifies `impl Runtime`, than the implementers need to use `impl Runtime` as well. If we do that, the implementers lose the ability to _name_ the runtime's type. They probably don't need to do that... but I figured we might as well be flexible.